discovery-provider/src/api/v1/helpers.py

```diff
@@ -419,7 +419,7 @@ def extend_transaction_details(transaction_details):
     new_transaction_details["transaction_date"] = transaction_details[
         "transaction_created_at"
     ]
-    if "metadata" in new_transaction_details:
+    if "tx_metadata" in transaction_details:
```

Comment:
also fixes this bug, which I think was preventing the metadata from working in the AUDIO tx history table modals!

Reply:
fab